app/components/Comments/CommentsList.jsx

```diff
@@ -20,22 +25,29 @@ export class CommentsList extends React.PureComponent {
   }
 
   render() {
-    const { comments, className, header, replyingTo, nesting = 1 } = this.props
+    const { comments, className, header, statementID, replyingTo, isAuthenticated, loggedInUser, nesting = 1 } = this.props
     const displayedComments = this.getDisplayedComments()
 
     return (
       <div className={classNames('comments-list', className)}>
         {header && <CommentsListHeader header={header} />}
         <FlipMove enterAnimation="fade" leaveAnimation={false}>
-          {displayedComments.map((comment) => (
+          {comments.size > 0 ? displayedComments.map((comment) => (
             <CommentDisplay
               key={comment.id}
               comment={comment}
               nesting={nesting}
               replyingTo={replyingTo}
               setReplyToComment={this.props.setReplyToComment}
+            />)) : (
+            <CommentForm
+              statementID={statementID}
+              replyTo={replyingTo}
+              setReplyToComment={this.props.setReplyToComment}
+              user={isAuthenticated ? loggedInUser : null}
+              inciteToParticipate={this.extractCommentType(className)}
```

Comment:
Ideally `className` shouldn't be used for something else than styling. Rather than having a `extractCommentType` function, I would suggest passing a `type` or `commentsType` prop from `app/components/Statements/StatementComments.jsx` and use that instead.

Reply:
https://github.com/CaptainFact/captain-fact-frontend/pull/745/commits/7cb7236532ab5df9d474f9a342118b57aee38491 implement your comment :smile: